FreeRTOS/FreeRTOS
# Add Makefile project for RISC-V QEMU virtpc

Description
-----------
This patch adds simple demo directory for QEMU virtpc machine.

A demo just prints Tx/Rx message of queue to serial port, use no
other hardware and use only primary core (currently hart0).
Other cores are simply going to wfi state and execute nothing else.

Test Steps
-----------
  $ make
  $ qemu-system-riscv32 -nographic -machine virt -net none \
    -chardev stdio,id=con,mux=on -serial chardev:con \
    -mon chardev=con,mode=readline -bios none \
    -kernel demo.elf -smp cpus=2

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
